src/lib/mixpanel/types.ts

```diff
@@ -50,6 +50,14 @@ export enum MixPanelEvent {
   SnapInstalled = 'Snap Installed',
   StartAddSnap = 'Start Add Snap',
   NoActionableQuotes = 'No Actionable Quotes',
+  LpDepositConfirm = 'LP Deposit Confirm',
```

Comment:
We may also want to do some reporting on previews, though not a blocker here and happy to do it a a follow-up